File: PrimaMateria/PrimaMateria/Classes/XTRSwapableViewController.m
//
//  XTRSwapableViewController.m
//  PrimaMateria
//
//  Created by Jerry Porter on 11/30/11.
//  Copyright (c) 2011 xTrensa. All rights reserved.
//

@implementation XTRSwapableViewController

- (void)setupUI { }

#pragma mark - View Management Methods

- (BOOL) shouldAutorotateToInterfaceOrientation: (UIInterfaceOrientation) interfaceOrientation {
    return UIInterfaceOrientationIsLandscape(interfaceOrientation);
}

@end

File: PrimaMateria/PrimaMateria/Classes/XTRSwapableViewController.swift
//
//  XTRSwapableViewController.swift
//  PrimaMateria
//
//  Created by Jerry Porter on 4/20/16.
//  Copyright © 2016 xTrensa. All rights reserved.
//

@objc class XTRSwapableViewController : UIViewController {
    var element : XTRElement?

    // MARK: - Initialization Methods
    
    required init?(coder aDecoder: NSCoder) {
        super.init(coder: aDecoder)!
    }

    func setupUI() {
    }
    
    // MARK: - View Management Methods
    
    override func shouldAutorotate() -> Bool {
        return true
    }
    
    override func supportedInterfaceOrientations() -> UIInterfaceOrientationMask {
        return [UIInterfaceOrientationMask.LandscapeLeft, UIInterfaceOrientationMask.LandscapeRight]
    }
    
    // MARK: - Memory Management Methods
    
}